_episodes/19-access-data.md

```diff
@@ -172,50 +196,48 @@ print(item.properties)
 {: .language-python}
 
 ~~~
-2022-01-25 10:56:17+00:00
-{'type': 'Polygon', 'coordinates': [[[6.071664488869862, 52.22257539160586], [4.464995307918359, 52.25346561204129], [4.498475093400055, 53.24019917467795], [6.1417542968794585, 53.20819279121764], [6.071664488869862, 52.22257539160586]]]}
-{'datetime': '2022-01-25T10:56:17Z', 'platform': 'sentinel-2b', 'constellation': 'sentinel-2', 'instruments': ['msi'], 'gsd': 10, 'view:off_nadir': 0, 'proj:epsg': 32631, 'sentinel:utm_zone': 31, 'sentinel:latitude_band': 'U', 'sentinel:grid_square': 'FU', 'sentinel:sequence': '0', 'sentinel:product_id': 'S2B_MSIL2A_20220125T105229_N0400_R051_T31UFU_20220125T151458', 'sentinel:data_coverage': 100, 'eo:cloud_cover': 100, 'sentinel:valid_cloud_cover': True, 'sentinel:processing_baseline': '04.00', 'created': '2022-01-25T20:59:17.398Z', 'updated': '2022-01-25T20:59:17.398Z'}
+2022-03-13 10:46:26+00:00
+{'type': 'Polygon', 'coordinates': [[[6.071664488869862, 52.22257539160586], [4.81543624496389, 52.24859574950519], [5.134718264192548, 52.98684040773408], [6.115758198408397, 52.84931817668945], [6.071664488869862, 52.22257539160586]]]}
+{'datetime': '2022-03-13T10:46:26Z', 'platform': 'sentinel-2b', 'constellation': 'sentinel-2', 'instruments': ['msi'], 'gsd': 10, 'view:off_nadir': 0, 'proj:epsg': 32631, 'sentinel:utm_zone': 31, 'sentinel:latitude_band': 'U', 'sentinel:grid_square': 'FU', 'sentinel:sequence': '0', 'sentinel:product_id': 'S2B_MSIL2A_20220313T103729_N0400_R008_T31UFU_20220313T140234', 'sentinel:data_coverage': 48.71, 'eo:cloud_cover': 99.95, 'sentinel:valid_cloud_cover': True, 'sentinel:processing_baseline': '04.00', 'sentinel:boa_offset_applied': True, 'created': '2022-03-13T17:43:18.861Z', 'updated': '2022-03-13T17:43:18.861Z'}
 ~~~
 {: .output}
 
 
 > ## Exercise: Search satellite scenes using metadata filters
 > Search for all the available Sentinel-2 scenes in the `sentinel-s2-l2a-cogs` collection that satisfy the following
 > criteria:
-> - intersect a provided bounding box (use ±0.05 deg in lat/lon from the point: 52.37N 4.90E);
-> - have been recorded in 2021;
-> - have a cloud coverage smaller than 5%.
+> - intersect a provided bounding box (use ±0.01 deg in lat/lon from the previously defined point);
+> - have been recorded between 20 March 2020 and 30 March 2020;
+> - have a cloud coverage smaller than 10%.
 >
-> How many scenes are available? Save the search results in  GeoJSON format.
+> How many scenes are available? Save the search results in GeoJSON format.
 >
 > >## Solution
 > >
 > > ~~~
-> > from shapely.geometry import Point
-> > p = Point(lon, lat)
-> > bbox = p.buffer(0.05).bounds
+> > bbox = point.buffer(0.01).bounds
```

Comment:
Certain command such as bbox and query used in this exercise may not be known to all the participants. Could we maybe add hints to these commands or links?

Reply:
Good point, I will add a hint for the `query` argument. About the bbox with the buffer, this is just one way to do it - it is not a requirement and it could be fun to show it while showing the solution.